fv3core/docker/dependencies.Dockerfile

```diff
@@ -158,95 +158,3 @@ RUN git config --global http.sslverify false && \
     cd NCEPlibs && \
```

Comment:
Remove NCEPlibs too

Reply:
I thought we might use `fv3gfs-environment`, but it doesn't look like it. Deleted that whole stage.